src/mds/schedule/operatorStep.cpp

```diff
@@ -327,8 +327,7 @@ ApplyStatus ChangePeer::Apply(
 ApplyStatus ScanPeer::Apply(const CopySetInfo& originInfo,
                             CopySetConf *newConf) {
     // (1) copyset success on start/cancel scan
-    // NOTE: If lastScanSec > 0, it means copyset just finished scaning
-    if (IsStartScan() == originInfo.scaning || originInfo.lastScanSec > 0) {
+    if (IsStartScan() == originInfo.scaning) {
```

Comment:
The logic here is a bit abstract.

1. IsStartScan() => IsStartScanOp()
2. if (  IsStartScanOp() && originInfo.scaning ) ||
        ( IsCncelScanOp() && !originInfo.scaning) ）{}

Reply:
> The logic here is a bit abstract.
> 
> 1. IsStartScan() => IsStartScanOp()
> 2. if (  IsStartScanOp() && originInfo.scaning ) ||
>    ( IsCncelScanOp() && !originInfo.scaning) ）{}

done